<?xml version="1.0" encoding="utf-8"?>
<!-- EN-Revision: 3fc56d76de1007a3c4f60b143c60c071f32546fb Maintainer: mch Status: ready -->
<!-- Reviewed: no -->
<phpdoc:exceptionref xml:id="class.unexpectedvalueexception" xmlns:phpdoc="http://php.net/ns/phpdoc" xmlns="http://docbook.org/ns/docbook" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xi="http://www.w3.org/2001/XInclude">
 <title>Класс UnexpectedValueException</title>
 <titleabbrev>UnexpectedValueException</titleabbrev>

 <partintro>

  <!-- {{{ UnexpectedValueException intro -->
  <section xml:id="unexpectedvalueexception.intro">
   &reftitle.intro;
   <para>
    Выбрасывается исключение, если значение не совпадает с набором значений. Обычно
    это происходит, когда функция вызывает другую функцию и ожидает, что возвращаемое
    значение будет определённого типа или значением, не включая арифметические ошибки,
    или ошибки, связанные с буфером.
   </para>
  </section>
  <!-- }}} -->

  <section xml:id="unexpectedvalueexception.synopsis">
   &reftitle.classsynopsis;

<!-- {{{ Synopsis -->
   <classsynopsis>
    <ooclass>
     <classname>UnexpectedValueException</classname>
    </ooclass>

    <classsynopsisinfo>
     <ooclass>
      <classname>UnexpectedValueException</classname>
     </ooclass>

     <ooclass>
      <modifier>extends</modifier>
      <classname>RuntimeException</classname>
     </ooclass>
    </classsynopsisinfo>

    <classsynopsisinfo role="comment">&InheritedProperties;</classsynopsisinfo>
    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:partintro/db:section/db:classsynopsis/db:fieldsynopsis[preceding-sibling::db:classsynopsisinfo[1][@role='comment' and text()='&Properties;']]))">
     <xi:fallback/>
    </xi:include>

    <classsynopsisinfo role="comment">&InheritedMethods;</classsynopsisinfo>
    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[@role='Exception'])">
     <xi:fallback/>
    </xi:include>
    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.exception')/db:refentry/db:refsect1[@role='description']/descendant::db:methodsynopsis[@role='Exception'])">
     <xi:fallback/>
    </xi:include>
   </classsynopsis>
<!-- }}} -->

   </section>

 </partintro>

 <!-- If the exception has any methods you'd like to document use this
 &reference.extname.entities.classname;
 -->

</phpdoc:exceptionref>
<!-- Keep this comment at the end of the file
Local variables:
mode: sgml
sgml-omittag:t
sgml-shorttag:t
sgml-minimize-attributes:nil
sgml-always-quote-attributes:t
sgml-indent-step:1
sgml-indent-data:t
indent-tabs-mode:nil
sgml-parent-document:nil
sgml-default-dtd-file:"~/.phpdoc/manual.ced"
sgml-exposed-tags:nil
sgml-local-catalogs:nil
sgml-local-ecat-files:nil
End:
vim600: syn=xml fen fdm=syntax fdl=2 si
vim: et tw=78 syn=sgml
vi: ts=1 sw=1
-->
